packages/eas-cli/src/project/publish.ts

```diff
@@ -140,11 +143,60 @@ export async function buildUpdateInfoGroupAsync(assets: CollectedAssets): Promis
   return updateInfoGroup as UpdateInfoGroup;
 }
 
+export async function buildBundlesAsync({
+  projectDir,
+  inputDir,
+}: {
+  projectDir: string;
+  inputDir: string;
+}) {
+  const packageJSON = await readJSON(path.resolve(projectDir, 'package.json'));
+  if (!packageJSON) {
+    throw new Error('Could not locate package.json');
+  }
+  const isCLIInstalled =
+    !!packageJSON.devDependencies['expo-cli'] || !!packageJSON.dependencies['expo-cli'];
+  if (!isCLIInstalled) {
+    throw new Error(
+      'expo-cli is not installed. If you would like us to handle bundling your app, please install it with: yarn add -D expo-cli'
+    );
+  }
+
+  Log.withTick(`Building bundle with expo-cli...`);
+  const spawnPromise = spawnAsync('yarn', [
+    'expo-cli',
```

Comment:
```suggestion
    'expo',
```
Both commands exists, but `expo` is what people mostly use and what we document, so I think it's less obscure.